rpc/src/v1/helpers/light_fetch.rs

```diff
@@ -523,6 +523,45 @@ where
 		}))
 	}
 
+	pub fn light_cull(&self, txq: Arc<RwLock<TransactionQueue>>) -> impl Future <Item = (), Error = Error> + Send {
+		let senders = txq.read().queued_senders();
+		if senders.is_empty() {
+			return Either::B(future::err(errors::no_light_transactions()));
```

Comment:
This is very unlikely to happen but it could happen if there is a race with calls to `eth_getTransactionByHash` i.e, a `cull` finished just between the transaction_by_hash and light_cull.

Then don't do a needless network request

Reply:
I think you should leave this is a comment within the code, just in case